src/grow_worker/worker.py

```diff
@@ -113,7 +113,7 @@ def kill_pool(pool: multiprocessing.pool.Pool) -> None:
 
 
 def grow_worker_task(
-    input_esdl: str, params_dict: ParamsDict, update_progress_handler: UpdateProgressHandler
+    input_esdl: str, workflow_config: ProtobufDict, update_progress_handler: UpdateProgressHandler
```

Comment:
Why updated to `ProtobufDict` type?

Reply:
Thanks for the explanation!  
The parsing could also be done by the orchestrator, the workflow definition contains the expected keys and types. Then we would get quicker feedback. The default value is also part of this config, so it doesn't have to be filled in by the worker.